### PR TITLE
BazelCI changes

### PR DESCRIPTION
**Pull request by dgkutnic**
I'll include a full list of changes once this PR is ready

### CODE REVIEW

File: .bazelci/presubmit.yml [hunk 1]
```diff
@@ -3,31 +3,49 @@ bazel: 0.28.1
...
   ubuntu1604:
     name: ":linux:"
-    shell_commands:
-    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
-    build_targets:
-    - "//plaidml/keras:wheel"
-    build_flags:
+    shell_commands: |
+      curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
```
Comment (flaub) [inline .bazelci/presubmit.yml]: Wrong OS here

File: .bazelci/presubmit.yml [hunk 2]
```diff
@@ -4,11 +4,23 @@ tasks:
...
+    - wget https://github.com/intel/compute-runtime/releases/download/19.35.13977/intel-igc-opencl_1.0.11-2500_amd64.deb
+    - wget https://github.com/intel/compute-runtime/releases/download/19.35.13977/intel-opencl_19.35.13977_amd64.deb
+    - wget https://github.com/intel/compute-runtime/releases/download/19.35.13977/intel-ocloc_19.35.13977_amd64.deb
+    - dpkg -i *.deb
+    - cd ..
+    environment:
+      PATH: /var/lib/buildkite-agent/miniconda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/go1.12.6.linux-amd64/bin:/opt/swift-4.2.1-RELEASE-ubuntu16.04/usr/bin
+      PLAIDML_DEVICE_IDS: llvm_cpu.0
+      PLAIDML_EXPERIMENTAL: 1
+      PLAIDML_TELEMETRY: 0
```
Comment (flaub) [inline .bazelci/presubmit.yml]: BTW I think we can drop this one, this was a feature we had before merging with Intel.

File: .bazelci/presubmit.yml [hunk 3]
```diff
@@ -4,11 +4,23 @@ tasks:
...
     shell_commands:
-    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
-    build_targets:
-    - "//plaidml/keras:wheel"
-    build_flags:
+    - curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh 
+    - sh ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+    - mkdir neo
+    - cd neo
```
Comment (flaub) [inline .bazelci/presubmit.yml]: I think we need the CPU drivers, not the NEO (GPU) ones.

File: .bazelci/presubmit.yml [hunk 4]
```diff
@@ -2,32 +2,49 @@
...
+    - wget https://github.com/intel/compute-runtime/releases/download/19.35.13977/intel-ocloc_19.35.13977_amd64.deb
+    - dpkg -i *.deb
+    - cd ..
+    environment:
+      PATH: /var/lib/buildkite-agent/miniconda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/go1.12.6.linux-amd64/bin:/opt/swift-4.2.1-RELEASE-ubuntu16.04/usr/bin
+      PLAIDML_DEVICE_IDS: llvm_cpu.0
+      PLAIDML_EXPERIMENTAL: 1
+      PLAIDML_TELEMETRY: 0
+    test_flags:
+    - "--config=linux_x86_64_bazelci"
```
Comment (flaub) [inline .bazelci/presubmit.yml]: Actually, I think we can do this:

```
- "--config=linux_x86_64"
- "--config=bazelci"
```

And then the `bazel.rc` could have: 
```
build:bazelci --build_tag_filters=-darwin,-metal,-msvc,-opencl,-pytorch
test:bazelci --test_tag_filters=-darwin,-metal,-msvc,-opencl,-pytorch
```

Although not sure how bazel handles combining multiple configs.

File: .bazelci/presubmit.yml (hunk 3 above)
Comment (dgkutnic) [reply .bazelci/presubmit.yml]: We've disabled the opencl tests in linux, but I'll keep this in mind if we decide to reenable them.

**Pull request merged**